### DISCUSSION TITLE
Task 07- Pronlem 11

### DISCUSSION THREAD

**Discussion by noreengao0924** [chulminy/CIVE497-CIVE700#175] [Q&A]
HI, I took my photos with iphone. However, the undistorted images generated by the given code are black.
![IMG_0003](https://github.com/chulminy/CIVE497-CIVE700/assets/64117230/6bb64977-3075-4622-a090-8c197354bd4c)

**Comment by chulminy** [comment]
I think your visualsfm model fails to reconstruct the model. Show your visuasfm screenshot

**Reply by noreengao0924** [reply]
![image](https://github.com/chulminy/CIVE497-CIVE700/assets/64117230/37b76346-82b4-421a-b5bb-9e8125486bc5)
This is my visualsfm screenshot.

**Comment by chulminy** [comment]
I think you use different images. A number of points is too small. Take photo from outside

**Reply by noreengao0924** [reply]
Thank you. I will try.
